### PR TITLE
the doors now flickers when you click on it

### PR DESCRIPTION
**Pull request by TermService**
## About The Pull Request
изображение

## Why It's Good For The Game

idk

## Changelog
:cl:
fix: The door flickers if you do not have access but you click on it
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]

**Review by MadmanMartian — CHANGES_REQUESTED** [review]
Check if attack delay works for this, else you'll have somebody buzzing up a storm at your door.

**Comment by TermService** [comment]
> 
> 
> Check if attack delay works for this, else you'll have somebody buzzing up a storm at your door.

This works the same as when you walk through the door.

**Comment by MadmanMartian** [comment]
And walk has a natural cooldown per-step. Raw attack_hand may not.

**Review by mattroks101 — APPROVED** [review]

**Comment by TermService** [comment]
oh, I completely forgot about

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -197,20 +197,21 @@
 	take_damage(damage)
 	return
 
-/obj/machinery/door/attack_hand(mob/user as mob)
-	if(src.allowed(user) && operable())
-		if(src.density)
+/obj/machinery/door/attack_hand(mob/user)
+	if(allowed(user) && operable())
```
Comment (TermService) [inline code/game/machinery/doors/door.dm:201]:
```suggestion
	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
	if(allowed(user) && operable())
```

**Comment by TermService** [comment]
> 
> 
> And walk has a natural cooldown per-step. Raw attack_hand may not.

done

**Review by TorinoFermic — APPROVED** [review]

**Review by MadmanMartian — APPROVED** [review]

**Pull request merged**